Automattic/vip-container-images
# Restore Gmagick extension for PHP 8.0 and 8.1

This PR adds Gmagick extension for PHP 8.0 and 8.1

For 8.0 it was reasonably simple after a bit of a trial and error: I had to add edge/testing channel.
For 8.1 it got a bit trickier, I had to to install it via `pecl` which brought in tons of dependencies. A side-effect is that build is a bit longer now. 

Steps to test locally:

1. Checkout the PR
2. From the root of the repo: `docker build -f php-fpm/Dockerfile.81 ./php-fpm`, verify image build and note the sha
3. `docker run -it $SHA sh`
4. `php -i | grep gmagick` should return results

To test further:

```
/tmp # php -a
Interactive shell

php > $i = new Gmagick('/tmp/test.png');
php > $i->thumbnailimage(100, 0);
php > $i->write('/tmp/cropthat.png');
```

Addresses #386
